dotnet/DexAgent/DexAgent/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Teams.Plugins.AspNetCore.DevTools.Extensions;
+using Microsoft.Teams.Plugins.AspNetCore.Extensions;
+using Microsoft.Teams.Common.Storage;
+using Microsoft.Teams.Apps;
+
+using DexAgent;
+using DexAgent.Interfaces;
+using DexAgent.Controllers;
+using Microsoft.Extensions.Options;
+
+var builder = WebApplication.CreateBuilder(args);
+
+var appBuilder = App.Builder()
+    .AddLogger(level: Microsoft.Teams.Common.Logging.LogLevel.Debug)
+    // The name of the auth connection to use.
+    // It should be the same as the OAuth connection name defined in the Azure Bot configuration.
+    .AddOAuth("github");
```

Comment:
Were you able to test out all the flows to verify the webhooks, cards, flows, etc work the same?